src/vaishnaviVJun20_Coding_Exams/Rotation.java

```diff
@@ -0,0 +1,39 @@
+package vaishnaviVJun20_Coding_Exams;
+
+/* Coding test 10 :
+
+Print minimum rotation required to compare both strings equality.
+
+Example 1  : 
+String input1 = "abcdef";
+String input2 = "defabc";
+output : 3
+
+Example 2  : 
+String input1 = "adbcdef"; -> push right
+fadbcde
+efadbcd
+defadbc
+String input2 = "defadbc";
+output : 3  */
+
+public class Rotation {
+	
+	void findMinimunRotation(String input1, String input2) {
+		int rotation=0;
+		 while(!input1.equals(input2)) {
```

Comment:
what if String can't be made equals even after all possible rotation ?? this will run infinitely ...

you can handled it with one more condition.
while(!input1.equals(input2) && count< input.length)

Reply:
Thank you ! @KrishnaKTechnocredits  
I will add this condition in my practice programs :)